posts/2024-05-01-spring-boot-3.adoc

```diff
@@ -111,17 +111,23 @@ mkdir -p src/main/liberty/config/
 cp target/liberty/wlp/usr/servers/defaultServer/server.xml src/main/liberty/config/server.xml
 ----
 +
-3. Add application arguments to your Spring Boot application configuration in the Liberty `server.xml` file.
+3. Configure the context root in your Spring Boot `application.properties` file.
++
+[source,properties]
+----
+server.servlet.context-path=/testpath
+----
++
+4. You can also configure the application using `server.xml`.  To illustrate, we'll set a Spring Boot property as an application argument in the Liberty `server.xml` configuration file.
```

Comment:
```suggestion
4. You can also configure the application in the Liberty `server.xml` file by setting a Spring Boot property as an application argument. The following example sets the logging level for the application.
```

Reply:
I like this overall and made a very minor tweak on top of it.